spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/Train.scala

```diff
@@ -67,17 +67,12 @@ object Train {
         curModel
       }
 
-      val state = if (param.stateSnapshot.isDefined) {
-        T.load(param.stateSnapshot.get)
+      val optimMethod = if (param.stateSnapshot.isDefined) {
+        OptimMethod.load[Float](param.stateSnapshot.get)
```

Comment:
I'm curious what parameters are stored in state snapshot now? Only the optimMethod hyper params? If there's (or will be) any options other than hyper parameters, would it be better if the state has a method like "getHyperParams"?